docs/pages/core_lib_doc/Data Layers/1.data_layers.md

```diff
@@ -45,7 +45,19 @@ class User(SoftDeleteMixin, Base):
     ...
 ```
 
-
+#### SQLAlchemy Structure:
+- <b>Imports:</b>
+    - Base: This is typically a SQLAlchemy [declarative_base()](https://docs.sqlalchemy.org/en/14/orm/mapping_api.html#sqlalchemy.orm.declarative_base) class. It's a base class for all your entities/model classes.
+    - SoftDeleteMixin: This is a custom mixin that adds soft delete functionality to your model, meaning instead of physically deleting records from the database, it marks them as deleted.
+    - Column, Integer, VARCHAR: These are classes and types provided by SQLAlchemy for defining columns in your database tables.
+- <b>Class Definition:</b>
+    - User: This is the model class or entity being defined [[1]](https://docs.sqlalchemy.org/en/14/orm/mapping_styles.html#declarative-mapping).
+    - SoftDeleteMixin, Base: SoftDeleteMixin is the custom class provided by core-lib. "SoftDeleteMixin first parameter ensures that methods from SoftDeleteMixin take precedence in case of conflicts between methods from the Base class".
+- <b>Table Configuration:</b>
+    - __tablename__: This specifies the name of the database table associated with this model. In the above case, it's set to 'user'.
```

Comment:
no need for 
- 
and then under it another 

hierarchy bolts are redundant
make sure to follow this in the entire docs